amplify/src/main/java/com/github/stkent/amplify/prompt/DefaultLayoutQuestionView.java

```diff
@@ -169,11 +170,20 @@ private void setButtonViewBackground(
         final Drawable buttonBackgroundDrawable
                 = getButtonBackgroundDrawable(fillColor, borderColor, borderWidthPx, cornerRadiusPx);
 
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN) {
+
+        if (android.os.Build.VERSION.SDK_INT <= 17) {
```

Comment:
Is there a reference you can point me to to explain why this change is necessary? I see the issue on my API level 16 AVD, but I'm struggling to find and reference to changes in API levels 16/17/18 that explain why this workaround is necessary.

Thanks!